docs/index.md

```diff
@@ -1,44 +1,72 @@
+     The intro must explain the overall process for feeding data to pulse
+     (essentially, customers must send events with data at several stages of their
+     software development workflow) -->
 
 The API key identifies your organization and authorizes you to push events to Pulse.
 
-## Pushing Data
-
 Currently, we support two ways to push data: a CLI and an HTTP API.
 
-### CLI
+## 1. Installing the Pulse CLI
 
-#### Install
+<!-- TODO
+     Clean up and simplify the installation procedure -->
 
-The CLI is distributed in a binary format (for multiple operating systems and architectures)
-that you can download:
+The CLI is distributed in a binary format (for multiple operating systems and architectures).
 
-```sh
-curl -fsSL "https://dl.bintray.com/codacy/pulse/event-cli/<VERSION>/<ARCH>/pulse-event-cli" -o event-cli
-```
+1.  Download the Pulse CLI:
 
-Use the latest version to replace `<VERSION>`: [ ![Download](https://api.bintray.com/packages/codacy/pulse/event-cli/images/download.svg)](https://bintray.com/codacy/pulse/event-cli/_latestVersion)
+    ```sh
+    curl -fsSL "https://dl.bintray.com/codacy/pulse/event-cli/<VERSION>/<ARCH>/pulse-event-cli" -o event-cli
+    ```
 
-Supported operating systems and architectures to replace `<ARCH>`:
+    Use the latest version to replace `<VERSION>`: [ ![Download](https://api.bintray.com/packages/codacy/pulse/event-cli/images/download.svg)](https://bintray.com/codacy/pulse/event-cli/_latestVersion)
 
--  pulse-event-cli_darwin_amd64
--  pulse-event-cli_linux_386
--  pulse-event-cli_linux_amd64
--  pulse-event-cli_windows_386
--  pulse-event-cli_windows_amd64
+    Supported operating systems and architectures to replace `<ARCH>`:
 
-Don't forget to make the binary executable and check that it works:
+    -  pulse-event-cli_darwin_amd64
+    -  pulse-event-cli_linux_386
+    -  pulse-event-cli_linux_amd64
+    -  pulse-event-cli_windows_386
+    -  pulse-event-cli_windows_amd64
 
-```sh
-chmod +x event-cli && \
-./event-cli help
-```
+2.  Make the binary executable and check that it works:
+
+    ```sh
+    chmod +x event-cli && \
+    ./event-cli help
+    ```
+
+### Using the HTTP API
+
+<!-- TODO
+     Figure out what is the best place to introduce the HTTP API as an alternative to the CLI -->
+
+Since it might be difficult to use the CLI to send some types of events (like changes or incidents),
+we also support pushing events to an HTTP POST endpoint.
+
+This might be useful for sending data from providers that only support webhooks. You can set up your provider to send a webhook to the following URL:
+
+`https://ingestion.acceleratedevops.net/v1/ingestion/<provider>?api_key=<API-KEY>`
 
-#### Run
+There is no specified format for events sent to this endpoint. Although, make sure to include all the fields documented above. If you are planning to push events this way, please let us know. Data will not be immediately available in your dashboard as we'll need to process it on our side.
```

Comment:
```suggestion
There is no specified format for events sent to this endpoint. Although, make sure to include all the fields documented below. If you are planning to push events this way, please let us know. Data will not be immediately available in your dashboard as we'll need to process it on our side.
```

Reply:
Good catch! :detective: Although this is still a work in progress (I forgot to set the pull request as Draft, sorry) and I will most probably get rid of the sentence "pointing" up or down. :smile:

Reply:
I thought that after noticed some TODOs 😁 
Looking good though